src/Void/Init.fs

```diff
@@ -5,6 +5,19 @@ open Void.Lang.Interpreter
 open Void.Lang.Editor
 open Void.ViewModel
 
+type VoidOptions = {
+    enableVerboseMessageLogging : bool
+}
+
+module Options =
+    let verboseMessagesFlag = "--verbose-message-logging"

```

Comment:
So same comment as before: I'm happy to merge it like this.

However, I wonder if the settings feature of .NET would be better for something like this. I say that because:

 1. Given my idea of a `CommandLineArgumentParserService`, it would work best if things like flipping in and out of debug mode were not command-line arguments. But again, for now, that's speculative.
 2. It would allow us to localize the logic that looks for this stuff rather than stringing it through. The class the needs it can just look at `Settings.Default...` or whatever that syntax is and we don't have to inject it from the top down.
 3. This is not something that users are going to want to do much, so hiding it away a little more can't hurt.